SPECS/fish/fish.spec

```diff
@@ -28,7 +28,7 @@ Distribution:   Mariner
 #   - user_doc/html/_static/underscore.js
 License:        GPLv2 and BSD and ISC and LGPLv2+ and MIT
 URL:            https://fishshell.com
-Source0:        https://github.com/fish-shell/fish-shell/releases/download/%{version}/%{name}-%{version}.tar.gz
+Source0:        https://github.com/fish-shell/fish-shell/releases/download/%{version}/%{name}-%{version}.tar.xz
```

Comment:
I think we're missing a signature update.

Reply:
Oops, copied the updated file the wrong way... fixed now.